gp-nested-forms/gpnf-auto-attach-child-files.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Gravity Perks // Nested Forms // Auto-attach Uploaded Files from Child to Parent Notifications
+ * https://gravitywiz.com/documentation/gravity-forms-nested-forms/
+ *
+ * This snippet automatically attaches files uploaded onto the child form to the parent form notifications.
```

Comment:
Let's include some instructions here on how this works. If parent form has any File Upload fields, this snippet will rely on the "Attachments" setting for each parent notification. Otherwise, you must specify a list of notifications by ID to which child uploads should be attached.